### PR TITLE
legion anvil micro buff

### PR DESCRIPTION
**Pull request by hako01**
## About The Pull Request

buffed the legion anvil

## Why It's Good For The Game

the good thing is, legion will more likely use melee
the bad thing is, the legion will more likely use melee

## Changelog
anvil.dm tweaked:
/obj/structure/anvil/obtainable/legion AKA legion fortress anvil to have maximum item quality of 10.

### CODE REVIEW

**Comment by hako01** [comment]
Cock and ball torture From Wikipedia, the free encyclopedia at en.wikipedia.org

Cock and ball torture (CBT) is a sexual activity involving application of pain or constriction to the male genitals. This may involve directly painful activities, such as genital piercing, wax play, genital spanking, squeezing, ball-busting, genital flogging, urethral play, tickle torture, erotic electrostimulation or even kicking. The recipient of such activities may receive direct physical pleasure via masochism, or knowledge that the play is pleasing to a sadistic dominant.

**Comment by SigholtStarsong** [comment]
Legion weapons already have better overall stats than the equivalents on a standard anvil. A war-honed axe at 8 quality can literally three tap people.

This isn't a microbuff, this is a significant balance adjustment.

**Comment by hako01** [comment]
dunno bro for such a "three tap kill" item it's rarely being used.

**Comment by Spc-Dragonfruits** [comment]
> Legion weapons already have better overall stats than the equivalents on a standard anvil. A war-honed axe at 8 quality can literally three tap people.
> 
> This isn't a microbuff, this is a significant balance adjustment.

Masterwork axes are universally busted. You only see the war-honed axe, because only the Legion spawns with a forge ready to go.

**Comment by Spc-Dragonfruits** [comment]
Furthermore,
This is completely unnecessary. **HARD** no from me. The last thing the Legion needs is better melee - it's what they excel at and it's what their faction is focused around, meaning they already have every single melee upside imaginable. From armor, to perks, to availability.

File: code/modules/smithing/anvil.dm [hunk 1]
```diff
@@ -300,7 +300,7 @@
 	desc = "A solid steel anvil with a stamped bull on it."
 	icon_state = "legvil"
 	anvilquality = 1
-	itemqualitymax = 8
+	itemqualitymax = 10
```
Comment (ProbablyCarl) [inline code/modules/smithing/anvil.dm:303]: This is an absurd buff. It doesn't do what you think it does, if you think this is a minor change.

**Comment by projectkepler-ru** [comment]
This sound terrible...

**Pull request closed without merging**